File: src/main/java/nl/martijnklene/api/application/entity/BlogPost.java
package nl.martijnklene.api.application.entity;

import javax.persistence.Entity;
import javax.persistence.Id;
import java.io.Serializable;
import java.util.UUID;

@Entity
public class BlogPost implements Serializable{
    @Id
    private UUID id;
    private String title;
    private String content;
    private String tags;
    private String author;

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public String getTags() {
        return tags;
    }

    public void setTags(String tags) {
        this.tags = tags;
    }

    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }
}

File: src/main/kotlin/nl/martijnklene/api/application/model/BlogPost.kt
package nl.martijnklene.api.application.model

import java.time.ZonedDateTime
import java.util.*

data class BlogPost(
    var id: UUID,
    var title: String,
    var content: String,
    var tags: String?,
    var author: String,
    var publishedAt: ZonedDateTime?,
    var createdAt: ZonedDateTime
)
